pillar.example

```diff
@@ -85,6 +88,29 @@ vhosting:
         #    http2: True
         #    letsencrypt: True
 
+        # Pagespeed example
+        #pagespeed-example.com:
+        #  pagespeed:
+        #    enable: True
+
+        # Pagespeed example with different cache path
+        #pagespeed-example-path.com:
+        #  pagespeed:
+        #    enable: True
+        #    file_cache_path: '/tmp/pagespeed'
+
+        # Pagespeed example with specific config
+        #pagespeed-example-config.com:
+        #  pagespeed:
+        #    enable: True
+        #    config:
+        #      DisableFilters: 'rewrite_images,combine_css'
```

Comment:
Would it be possible to declare this as an array? since it is one :see_no_evil:

Reply:
Yes, I did have this before but found out that there are about 50 config keys. Handling them all is horrible, so therefor its inserting this rawly into the pagespeed section.